.repo-metadata.json

```diff
@@ -2,12 +2,12 @@
   "name": "bigqueryconnection",
   "name_pretty": "Google BigQuery Connection",
   "product_documentation": "https://cloud.google.com/bigquery/docs/reference/bigqueryconnection",
-  "client_documentation": "https://googleapis.dev/nodejs/bigqueryconnection/latest",
+  "client_documentation": "https://googleapis.dev/nodejs/googleapis/latest/bigqueryconnection/index.html",
```

Comment:
That is not correct 🙃  This link is wrong right now because the module hasn't shipped yet, so the docs job hasn't run.  @steffnay any objections to shipping?

Reply:
@JustinBeckwith No, we can go ahead.